pkg/gofr/container/container.go

```diff
@@ -54,80 +54,126 @@ func NewContainer(conf config.Config) *Container {
 }
 
 func (c *Container) Create(conf config.Config) {
+	c.setAppName(conf)
+	c.setAppVersion(conf)
+	c.createLogger(conf)
+	c.createMetricsManager()
+	c.createRedis(conf)
+	c.createSQL(conf)
+	c.createPubSub(conf)
```

Comment:
This should not be included in this PR, as it is not directly related to masking logs. Please create a separate PR for this.